activity_browser/application.py

```diff
@@ -41,14 +41,14 @@ def deleteLater(self):
     # https://bugreports.qt.io/browse/QTBUG-85546
     # https://github.com/mapeditor/tiled/issues/2845
     # https://doc.qt.io/qt-5/qoperatingsystemversion.html#MacOSBigSur-var
-    supported = {"10.10", "10.11", "10.12", "10.13", "10.14", "10.15"}
+    supported = {"10.10", "10.11", "10.12", "10.13", "10.14", "10.15", "13.6"}
     if QSysInfo.productVersion() not in supported:
         os.environ["QT_MAC_WANTS_LAYER"] = "1"
         os.environ["QTWEBENGINE_CHROMIUM_FLAGS"] = "--disable-gpu"
         log.info("Info: GPU hardware acceleration disabled")
 
-if QSysInfo.productType() in ["arch", "nixos"]:
-    os.environ["QTWEBENGINE_CHROMIUM_FLAGS"] = "--no-sandbox"
+if QSysInfo.productType() in ["arch", "nixos", "osx"]:
+    os.environ["QTWEBENGINE_CHROMIUM_FLAGS"] = "{} --no-sandbox".format(os.getenv("QTWEBENGINE_CHROMIUM_FLAGS"))
```

Comment:
This needs a bit of explanation on why and how to set the envvar.

Reply:
added the why